stable/bookstack/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.24.2
 description: BookStack is a simple, self-hosted, easy-to-use platform for organising and storing information.
 name: bookstack
-version: 0.1.2
+version: 0.1.3
```

Comment:
this is enough of a functional change to bump a minor version.   in fact since this is in stable, you should probably bump to `1.0.0`.

Reply:
I am no the author of the chart, so I'm not really able to validate if it's ready for 1.0.0, or if it's such a major change if it only exposes something that was available underneath, in the image anyway. 

To me, adding a mapping between `values.yml` and env values felt minor.

While bumping it to `0.2.0` does make sense after you mentioned the functional character of the change, I don't think I should be the one to push bump to 1.0.0 after such a small change :)